src/theme/form-field.scss

```diff
@@ -115,6 +115,7 @@
       }
 
       .mat-form-field-subscript-wrapper {
+        width: 125%;
```

Comment:
what's this? 125% ?

Reply:
We needed to fit the hint text into 2 lines. For this  we had to enable the hint to go beyond the 100% width of the input.